packages/samples/headless-ssr/src/app/react/components/facets.tsx

```diff
@@ -0,0 +1,26 @@
+import {FacetState, Facet as FacetController} from '@coveo/headless';
```

Comment:
Should these import from headless/ssr (although they are the same technically)?

Reply:
woops, you're right!

Reply:
after all, I'm going to copy almost all the exports from `@coveo/headless` in `@coveo/headless/ssr`.